website/server/libs/cron.js

```diff
@@ -347,7 +393,7 @@ export function cron (options = {}) {
 
   // Login Incentives
   user.loginIncentives++;
-  user.addNotification('LOGIN_INCENTIVE', {});
+  awardLoginIncentives(user);
```

Comment:
I see what you're saying, I think the reason why I put the user saving after the tasks processing was because if the user is saved, saying that cron was done, the tasks won't be processed anymore. Unfortunately these bugs will keep happening, we just have to decide which part is more important